VIS/Areas/VIS/Scripts/app/controls.js

```diff
@@ -2499,8 +2499,45 @@
             }
 
         };
+        if (displayType == VIS.DisplayType.Search) {
+            $ctrl.visautocomplete({
+                source: function (term, response) {
+                    var sql = self.lookup.info.queryAll;
+                    var keyColumn = self.lookup.colName;
+                    var displayColumn = self.lookup.info.displayColSubQ;    
 
+                    sql = sql.replace(displayColumn, displayColumn + ' finalValue');
```

Comment:
insert 'AS' for alias, may there is need to wrap display column with Brackets ().